tidb-lightning/tidb-lightning-logical-import-mode-usage.md

```diff
@@ -49,12 +49,12 @@ TiDB Lightning 的完整配置文件可参考[完整配置及命令行参数](/t
 
 冲突数据是指两条或两条以上记录中存在主键或唯一键列数据重复。TiDB Lightning 的逻辑导入模式通过 [`conflict.strategy`](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-任务配置) 配置冲突数据的处理行为，使用不同的 SQL 语句进行导入。
 
-| 策略 | 冲突时默认行为 | 对应 SQL 语句 |
-|:---|:---|:---|
-| `"replace"` | 新数据替代旧数据 | `REPLACE INTO ...` |
-| `"ignore"` | 保留旧数据，忽略新数据 | `INSERT IGNORE INTO ...` |
-| `"error"` | 终止导入 | `INSERT INTO ...` |
-| `""` | 不进行冲突检查和处理，但如果存在有主键和唯一键冲突的数据，会在后续步骤报错  | 无 |
+| 策略 | 冲突时默认行为          | 对应 SQL 语句 |
+|:---|:-----------------|:---|
+| `"replace"` | 新数据替代旧数据         | `REPLACE INTO ...` |
+| `"ignore"` | 保留旧数据，忽略新数据      | `INSERT IGNORE INTO ...` |
+| `"error"` | 终止导入             | `INSERT INTO ...` |
+| `""` | 会被转换为 `"error"`，终止导入 | 无 |
```

Comment:
```suggestion
| `""` | 会被转换为 `"error"`，遇到冲突记录时终止导入 | 无 |
```

Reply:
this comment is from Frank:https://github.com/pingcap/docs-cn/pull/16688#discussion_r1522445755

Reply:
all modified